core/kernel/pseudo_ta.c

```diff
@@ -261,6 +261,12 @@ static TEE_Result verify_pseudo_tas_conformance(void)
 	const struct pseudo_ta_head *end =
 		SCATTERED_ARRAY_END(pseudo_tas, struct pseudo_ta_head);
 	const struct pseudo_ta_head *pta;
+	uint32_t pth_size = sizeof(struct pseudo_ta_head);
+	uint64_t align_start = (uint64_t)start;
+
+	/* make start align to sizeof(struct pseudo_ta_head) byte */
+	align_start = ROUNDUP(align_start, pth_size);
+	start = (const struct pseudo_ta_head *)align_start;
```

Comment:
Since the struct pseudo_ta_head is 128 bytes aligned, why do we still need to change this function implementation?